### PR TITLE
[FAB-18098] Update 1.4 docs for devmode

### PR DESCRIPTION
**Pull request by DereckLuo**
Signed-off-by: Chongxin Luo <Chongxin.Luo@ibm.com>

#### Type of change

- Documentation update

#### Description

Documentation updates for `Using dev mode`.

#### Related issues

[FAB-18098](https://jira.hyperledger.org/browse/FAB-18098)

### CODE REVIEW

File: docs/source/peer-chaincode-devmode.rst [hunk 1]
```diff
@@ -14,6 +14,23 @@ single running instance can be accessed from multiple channels.
...
+::
+    make orderer; make peer; make configtxgen;
+::
+
+Setup PATH environment variable to include orderer and peer binaries.
+::
+    cd /.build/bin/; export=$PATH:$(pwd)
+::
+
+Setup FABRIC_CFG_PATH to point at sampleconfig.
```
Comment (wlahti) [inline docs/source/peer-chaincode-devmode.rst:29]:
```suggestion
Setup FABRIC_CFG_PATH to point to the sampleconfig directory.
```

File: docs/source/peer-chaincode-devmode.rst [hunk 2]
```diff
@@ -14,6 +14,23 @@ single running instance can be accessed from multiple channels.
 
 All commands are executed from the ``fabric`` folder.
 
+Setup environment
+-----------------
+Build binaries for orderer, peer and configtxgen.
+::
+    make orderer; make peer; make configtxgen;
```
Comment (wlahti) [inline docs/source/peer-chaincode-devmode.rst:21]: Can't this just be done in a single call? 

`make orderer peer configtxgen`

File: docs/source/peer-chaincode-devmode.rst (hunk 1 above)
Comment (stephyee) [reply docs/source/peer-chaincode-devmode.rst:29]: 🛠️

File: docs/source/peer-chaincode-devmode.rst (hunk 2 above)
Comment (stephyee) [reply docs/source/peer-chaincode-devmode.rst:21]: 🛠️

File: docs/source/peer-chaincode-devmode.rst [hunk 3]
```diff
@@ -14,6 +14,23 @@ single running instance can be accessed from multiple channels.
...
+::
+
+Setup PATH environment variable to include orderer and peer binaries.
+::
+    cd /.build/bin/; export PATH=$PATH:$(pwd)
+::
+
+Setup FABRIC_CFG_PATH to point at sampleconfig.
+::
+    cd /sampleconfig/; export FABRIC_CFG_PATH=$(pwd)
```
Comment (stephyee) [inline docs/source/peer-chaincode-devmode.rst]:
```suggestion
    cd sampleconfig/; export FABRIC_CFG_PATH=$(pwd)
```

Remove beginning `/`

Comment (DereckLuo) [reply docs/source/peer-chaincode-devmode.rst]: done.

**Comment by lindluni** [comment]
Will address in a different PR

**Pull request closed without merging**